managed/ui/src/components/config/Storage/StorageConfigurationContainer.js

```diff
@@ -30,6 +34,16 @@ const mapDispatchToProps = (dispatch) => {
       });
     },
 
+    setInitialConfigValues: (initialValues) => {
+      return dispatch(setInitialConfigValues(initialValues));
+    },
+
+    updateCustomerConfig: (configUUID) => {
```

Comment:
usage in `StorageConfiguration.js` doesn't match the definition - a method called with an object, not with just an id

Reply:
Fixed.